install.php

```diff
@@ -31,10 +31,10 @@
 $admin = new User();
 $admin->first_name = "admin";
 $admin->last_name = "user";
-$admin->email = "admin@concordia.ca";
+$admin->email = "admin";
 $admin->is_admin = 1;
 $admin->is_instructor = 0;
-$admin->password_digest = password_hash('root', PASSWORD_DEFAULT);
```

Comment:
I think the password he requested was admin no?

Reply:
yup checked its admin admin

Reply:
@drouba so all good right?